resources/js/src/app/store/modules/LastSeenModule.js

```diff
@@ -48,24 +62,28 @@ const actions =
             return null;
         },
 
-        getLastSeenItems({commit}, maxItems)
+        getLastSeenItems({commit})
         {
             if (!state.isLastSeenItemsLoading)
             {
                 return new Promise((resolve, reject) =>
                 {
-                    const params = {items: maxItems || App.config.itemLists.lastSeenNumber};
-
                     commit("setIsLastSeenItemsLoading", true);
 
-                    ApiService.get("/rest/io/item/last_seen", params, {keepOriginalResponse: true})
+                    ApiService.get("/rest/io/item/last_seen", {items: App.config.itemLists.lastSeenNumber}, {keepOriginalResponse: true})
```

Comment:
@stentrop War nicht geplant, die globale Einstellung für die Anzahl der zuletzt gesehenen Artikel zu entfernen und durch eine Einstellung pro Widget zu ersetzen? Falls ja, macht es hier keinen Sinn, genau diesen Parameter zu entfernen.

Reply:
ja genau

Reply:
Ich habe den Wert aus der Config nun als Deprecated markiert und er wird nichtmehr genutzt. Stattdessen Beschränkt der Parameter items in der Komponente nun die Daten die aus dem Vuex Store geladen werden.
Beim Schreiben von last_seen werden 20 Artikel gespeichert und ebenso 20 geladen per REST.